pyproject.toml

```diff
@@ -56,11 +56,11 @@ codespell = "^2.2.6"
 optional = true
 
 [tool.poetry.group.unit.dependencies]
-coverage = {extras = ["toml"], version = "^7.3.2"}
+coverage = {extras = ["toml"], version = "^7.4.0"}
 pydantic = "^1.10.13"
-pytest = "^7.4.3"
+pytest = "^7.4.4"
 pytest-mock = "^3.12.0"
-pytest-asyncio = "^0.21.1"
+pytest-asyncio = "*"
```

Comment:
We don't really need `pytest-asyncio` for unit tests (it suppresses warnings) so we should be fine with whatever poetry pulls (whichever version is pulled by `pytest-operator`)